src/js/main.js

```diff
@@ -90,3 +90,45 @@ const modified = new Swiper(".modified", {
     },
   },
 });
+
+// Адаптивное меню нижней шапки "Еще" //
+
+function ld() {
+  const a = window.innerWidth;
+  const b = window.innerHeight;
+  const c = window.outerWidth;
+  const d = window.outerHeight;
+  console.log(
+    "innerWidth = " +
+      a +
+      " " +
+      "innerHeight = " +
+      b +
+      " " +
+      "outerWidth = " +
+      c +
+      " " +
+      "outerHeight = " +
+      d
+  );
+}
+window.addEventListener("load", ld());
```

Comment:
Вторым аргументом должна быть сама функция, а не результат ее выполнения
```suggestion
window.addEventListener("load", ld);
```